### ISSUE TITLE
Column With Name "value" Not Showing Title in Filter's Selectbox

### ISSUE THREAD

**Issue by dantaeusb** [TablePlus/TablePlus-Windows#427]
1. Which driver are you using and version of it (Ex: PostgreSQL 10.0): MySQL 5.7-33-36 (Percona database)

2. Which TablePlus build number are you using (the number on the welcome screen, Ex: build 81): 172

3. The steps to reproduce this issue: 

- Open a table with col named "value"
- Right-click "value" col, "Filter with this column"
- Name will be hidden in selectbox, I believe this could be due to some kind of formatting/it's reserved word

![missing col name](https://user-images.githubusercontent.com/4356112/125472039-2297a517-1aa3-4340-b4f5-7751cc62ed4e.png)

Not a big deal, but a weird case. I've restarted TablePlus before submitting the issue to make sure it's reproduceable and not just a cache issue or something, so I believe it's a minor bug.

**Comment by son11592** [comment]
Hi @dantaeusb, thank you for you feedback. I have tried to reproduce this issue but nothing happened. Could you please give me a gif or video about this issue? It might help us to find out the problem. Thanks.

**Comment by dantaeusb** [comment]
Sure, here it goes. Maybe it's something about my environment, but sounds unlikely + I'm definitely on the latest version

https://user-images.githubusercontent.com/4356112/125688326-bb963cfe-f2e7-484e-a395-aa566e2c9636.mp4

**Comment by huyphams** [comment]
Hi @dantaeusb if you have a chance, please give install this patch and see how it goes: https://www.dropbox.com/s/eqztxlh2bai0lyd/TablePlusUpdate.exe?dl=0
Thanks

**Comment by dantaeusb** [comment]
Hi!

Looks like it helped: text didn't disappear when I tried this version, thank you.

I'm not sure if that's related to this exact version: but when I tested things I've just noticed that when I try to copy cell value either by using context menu ("Copy Cell Value") or with Ctrl+A, Ctrl+C, it does not copy anything after asterisk sign, including sign itself. Not sure if that was happening before. I've attached an example of varchar value in field and copied value, pasted to VSCode below:

![image](https://user-images.githubusercontent.com/4356112/125972532-47644587-d289-4ad7-a28e-4f63327fcfaf.png)
